### DISCUSSION TITLE
PRQL Style Guide?

### DISCUSSION THREAD

**Discussion by whoarusty** [PRQL/prql#2855] [Ideas]
Has there been any discussion of a PRQL Style Guide -- something like [SQL Style Guide](https://www.sqlstyle.guide/). I know each programming language has it's own set of guidelines, so I was wondering if this was discussed for PRQL and if so where to find it.

If one doesn't exist, I could fork the [SQL Style Guide](https://github.com/treffynnon/sqlstyle.guide/blob/gh-pages/_includes/sqlstyle.guide.md) and modify it for PRQL.

**Comment by max-sixty** [comment]
Thanks for the question!

We're developing this mostly by developing the autoformatter — check out https://github.com/PRQL/prql/issues/2757 for some links. I think implementing this in code will be easier to iterate on than docs (though no objection to describing the current rules, as long as someone is up for keeping them updated)

Running the autoformatter on queries (e.g. `prqlc fmt foo.prql`), finding where it produces bad results, and adding those as issues would be a nice contribution. 

There are some outstanding questions in that issue, such as when to break lines — opinions (with examples) are welcome...
